cardano-cli/src/Cardano/CLI/EraBased/Commands/Governance/Committee.hs

```diff
@@ -14,28 +20,50 @@ import           Cardano.CLI.Types.Key.VerificationKey
 import           Data.Text (Text)
 
 data GovernanceCommitteeCmds era
-  = GovernanceCommitteeKeyGenColdCmd
-      (ConwayEraOnwards era)
-      (File (VerificationKey ()) Out)
-      (File (SigningKey ()) Out)
-  | GovernanceCommitteeKeyGenHotCmd
-      (ConwayEraOnwards era)
-      (File (VerificationKey ()) Out)
-      (File (SigningKey ()) Out)
-  | GovernanceCommitteeKeyHashCmd -- TODO to be moved under the top-level command group "key"
-      (ConwayEraOnwards era)
-      AnyVerificationKeySource
-  | GovernanceCommitteeCreateHotKeyAuthorizationCertificateCmd -- TODO to be moved under the top-level command group "key"
-      (ConwayEraOnwards era)
-      (VerificationKeyOrHashOrFile CommitteeColdKey)
-      (VerificationKeyOrHashOrFile CommitteeHotKey)
-      (File () Out)
-  | GovernanceCommitteeCreateColdKeyResignationCertificateCmd
-      (ConwayEraOnwards era)
-      (VerificationKeyOrHashOrFile CommitteeColdKey)
-      (File () Out)
+  = GovernanceCommitteeKeyGenColdCmd                            (GovernanceCommitteeKeyGenColdCmdArgs                           era)
+  | GovernanceCommitteeKeyGenHotCmd                             (GovernanceCommitteeKeyGenHotCmdArgs                            era)
+  | GovernanceCommitteeKeyHashCmd                               (GovernanceCommitteeKeyHashCmdArgs                              era) -- TODO to be moved under the top-level command group "key"
+  | GovernanceCommitteeCreateHotKeyAuthorizationCertificateCmd  (GovernanceCommitteeCreateHotKeyAuthorizationCertificateCmdArgs era) -- TODO to be moved under the top-level command group "key"
```

Comment:
As discussed the TODO needs to be deleted

Reply:
Done!